Library/Homebrew/utils/ruby.sh

```diff
@@ -45,15 +56,29 @@ EOS
       then
         HOMEBREW_RUBY_PATH="/System/Library/Frameworks/Ruby.framework/Versions/Current/usr/bin/ruby"
       else
-        HOMEBREW_RUBY_PATH="$(type -P ruby)"
+        HOMEBREW_RUBY_PATH=$(type -P ruby)
+        if [[ $(test-ruby $HOMEBREW_RUBY_PATH) != "true" ]]
```

Comment:
@maxim-belkin  the test-ruby fn considers $1. 
i suspect this will be problematic. 
can i know the trick that handles spaced ruby path here?

Reply:
right -- here quotes are necessary. Will fix! Thanks again!

Reply:
Fixed in 083f56f. 
I was considering using `"$@"` but then decided to proceed with `$1`. Thanks for catching it.